packages/export-adapters/index.js

```diff
@@ -11,6 +12,8 @@ const exportAdapter = (blockData, exportFormat) => {
     return { data: JSON.stringify(blockData, null, 2), ext: 'json' };
   case 'txt':
     return { data: draftToTxt(blockData), ext: 'txt' };
+  case 'digitalpaperedit':
```

Comment:
Just a question - do we definitely want these export formats as lowercase? (as opposed to camelcase etc. which is adopted in the rest of the codebase)

Reply:
yeah, not sure, I guess now that would introduce a breaking change